docs/source/en/tutorials/fast_diffusion.md

```diff
@@ -34,15 +28,23 @@ Then upgrade the other required libraries too:
 pip install -U transformers accelerate peft
 ```
 
-To benefit from the fastest kernels, use PyTorch nightly. You can find the installation instructions [here](https://pytorch.org/). 
+Install [PyTorch nightly](https://pytorch.org/) to benefit from the latest and fastest kernels:
+
+```bash
+pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu
```

Comment:
No need to install `torchaudio` and `torchvision`. Moreover,  it installs the CPU variant. Below is the right command:

```bash
pip3 install --pre torch --index-url https://download.pytorch.org/whl/nightly/cu121
```